src/shapes/Cone.cpp

```
@@ -11,126 +12,129 @@ Raytracer::Cone::hits(const Raytracer::Ray &ray) const {
+
   if (final_t <= 0) {
-      return {0.0, _color, this};
+    return {0.0, _color, this};
   }
-  
+
   return {final_t, _color, this};
 }
 
-Math::Vector3D Raytracer::Cone::getNormal(const Math::Point3D &hit_point) const {
+Math::Vector3D Raytracer::Cone::getNormal(
```

Cppcheck (2.10-2)

[style] The function 'getNormal' is never used.

(unusedFunction)